docs/reference/bpmn-processes/error-events/error-events.md

```diff
@@ -3,49 +3,68 @@ id: error-events
 title: "Error events"
 ---
 
-Error events are events which reference an error. They are used to handle business errors in a process.
+In process automation, you may have to handle with errors related to the process, also known as business errors.
```

Comment:
```suggestion
In process automation, you may have to deal with errors related to the process, also known as business errors.
```

Reply:
I had this at some point because I wanted to introduce the term `error handling` earlier, but I struggled with the difference between handle and catch. I've talked with Philipp and we think it's a good idea to move away from the word handle altogether. I'll only leave it in the heading `Unhandled error` because the incident (due to not catching the error) has the unhandled error type. Everything else relates to catching which is also the term used by the bpmn spec.